planner/core/testdata/expression_rewriter_suite_out.json

```diff
@@ -7,10 +7,9 @@
         "Plan": [
           "HashJoin 2.25 root  inner join, equal:[eq(test.t1.a, test.t2.a) eq(test.t1.b, test.t2.b)]",
           "├─HashAgg(Build) 1.69 root  group by:test.t2.a, test.t2.b, funcs:firstrow(test.t2.a)->test.t2.a, funcs:firstrow(test.t2.b)->test.t2.b",
-          "│ └─TableReader 1.69 root  data:HashAgg",
```

Comment:
We can ignore this case since it will be solved by https://github.com/pingcap/tidb/issues/38874.